### ISSUE TITLE
RESTful webservice

### ISSUE THREAD

**Issue by johnny-morrice** [johnny-morrice/godelbrot#12]
The webservice described under johnny-morrice/webdelbrot#7 is to be moved to the godelbrot project.  

This will enable other clients to have a ready made frontend service for rendering, zooming, controlling concurrency etc.

**Comment by johnny-morrice** [comment]
We also ought to have a tiny CLI client for the rest service too.  Will merge into master once that is done and tested.

**Comment by johnny-morrice** [comment]
9c21fc3 was largely a lie.  It surely made a start on this, but progress towards a system that actually works is found in

f0e8ed5262c7cadf0f1b99ce4b753aabf86ee9d7

f9d309468f2689eb90416fb1a9177ab376ed2575

7f75fa1cc1b19c1a903720f30e9ce16eabff2456

7aa955e6f0b3a56c1c2df414c957a2aea4d752fe

4618a06e1bf075d419597baff72223966804813e

9f2b989cadf35919ce0853b844d21f14a1b6a1ba

3b4fff82c8467c4005ca5cc92338bef6dd093014

871f50127eb1c0c3a12beeeaa5721c1bafd9de8f

6a67249e96b165d4d2bca1304f23a233fcf6bbb5

**Comment by johnny-morrice** [comment]
Render task is crashing returning exit 1.  Should dump subprocess stderr to server log.

**Comment by johnny-morrice** [comment]
Webservice is now producing images.  Zooming needs testing.

Then integration test with (browser) client.

**Comment by johnny-morrice** [comment]
See commits

9d83db9

2269f2e

dd706ae

For more work

**Comment by johnny-morrice** [comment]
Done!

In one terminal run

```
$ configbrot | restfulbrot
```

In another run

```
$ configbrot | clientbrot > img/nozoom.png
$ configbrot | clientbrot --cycle --getrq CJRRiGU_neADTL-GWEoC -xmin 100 -xmax 350 -ymin 100 -ymax 280 > img/zoom.png
```

Where the hash above is replaced with data you see in the restfulbrot output.  (Or try `-debug` to see this in the client)

Commits

2a5bcf2

5577653

b4fd7d8

9a86162

6693b53

7a45500
